pkg/apis/pipeline/v1beta1/provenance_conversion.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2022 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"context"
+
+	v1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+)
+
+func (p Provenance) convertTo(ctx context.Context, sink *v1.Provenance) {
```

Comment:
This is to be kept for the completeness of PR function only.

Will be removed and rebased if https://github.com/tektoncd/pipeline/pull/5794 is merged first.